### DISCUSSION TITLE
Possible Feature Request - attach a video file with the recipe.

### DISCUSSION THREAD

**Discussion by aram535** [mealie-recipes/mealie#353] [Ideas]
I'm a very audio/visual person, I much use a video to learn something than to read it - recipes included.  Has there been any thought on how to include a video along with a recipes?  

I don't mean link to a youtube video, I have dozen of self-hosted videos that I captured while cooking in kitchens that I would like to include in the entry into the system.  

Obviously encoding and storing the video would not be advisable but it would be nice to be be to browse to a video file and attach it locally.

**Comment by hay-kot** [comment]
There's a lot that goes into properly streaming video and I think it makes more sense to host your videos on a platform that specializes in that and then link them in the markdown, similar to embedding a YouTube video. I'm not sure what the self-hosted alternatives are but I'm sure there's something that would get the job done.
